packages/commons/package.json

```diff
@@ -12,5 +12,7 @@
     "reset": "yarn clean"
   },
   "dependencies": {
+    "@types/uuid": "8.3.0",
+    "uuid": "^8.3.1"
```

Comment:
You can remove this. You can find the reason in my comment at `pet.class.ts` 

```suggestion
```

Reply:
fixed in c9562eec904cabada1f650c950ed9ae88aacd317